rio_alpha/alpha.py

```diff
@@ -3,17 +3,20 @@
 import riomucho
 from rio_alpha.alpha_mask import mask_exact
 
-# Convert window to an instance of rasterio.windows.Window
-# if possible, to avoid Rasterio deprecation warnings.
-#
-# The window_guard function is used like:
-#
-# window = window_guard(window)
+# Convert window to an instance of rasterio.windows.Window, if possible,
+# to avoid Rasterio deprecation warnings.
 try:
     from rasterio.windows import Window
-    window_guard = lambda x: x if isinstance(x, Window) else Window.from_slices(*x)
 except ImportError:
```

Comment:
The following two lines are uncovered, but it's unreasonably complicated to trick it into an ImportError in a test.